api/control/v1alpha1/types.go

```diff
@@ -105,30 +105,17 @@ type BindingPolicySpec struct {
 	// sets are combined by union.
 	Downsync []DownsyncPolicyClause `json:"downsync,omitempty"`
 
-	// WantSingletonReportedState means that for objects that are distributed --- taking
+	// WantSingletonReportedState means that for workload objects that are distributed --- taking
 	// all BindingPolicies into account --- to exactly one WEC, the object's reported state
 	// from the WEC should be written to the object in its WDS.
-	// WantSingletonReportedState connotes an expectation that indeed the object will
-	// propagate to exactly one WEC, but there is no guaranteed reaction when this
-	// expectation is not met.
+	// If any of the workload objects are distributed to more or less than 1 WEC then
```

Comment:
Is the rest of the comment makes sense for 0 WECs as indicated in the "less" here ?  How ?

Reply:
If the user expects the object to be distributed to 1 WEC and the actual number is 0 then this is a discrepancy to report.